src/GraphQL.Tests/Execution/ResolveFieldContextTests.cs

```diff
@@ -46,45 +46,51 @@ public void argument_returns_long()
         {
             long val = 1000000000000001;
             _context.Arguments["a"] = val;
-            var result = _context.Argument<long>("a");
+            var result = _context.GetArgument<long>("a");
             result.ShouldEqual(1000000000000001);
         }
 
         [Fact]
         public void argument_returns_enum()
         {
             _context.Arguments["a"] = SomeEnum.Two;
-            var result = _context.Argument<SomeEnum>("a");
+            var result = _context.GetArgument<SomeEnum>("a");
             result.ShouldEqual(SomeEnum.Two);
         }
 
         [Fact]
         public void argument_returns_enum_from_string()
         {
             _context.Arguments["a"] = "two";
-            var result = _context.Argument<SomeEnum>("a");
+            var result = _context.GetArgument<SomeEnum>("a");
             result.ShouldEqual(SomeEnum.Two);
         }
 
         [Fact]
         public void argument_returns_enum_from_number()
         {
             _context.Arguments["a"] = 1;
-            var result = _context.Argument<SomeEnum>("a");
+            var result = _context.GetArgument<SomeEnum>("a");
             result.ShouldEqual(SomeEnum.Two);
         }
 
         [Fact]
-        public void throw_error_if_argument_doesnt_exist()
+        public void argument_returns_default_when_missing()
         {
-            Expect.Throws<ExecutionError>(() => _context.Argument<string>("wat"));
+            _context.GetArgument<string>("wat").ShouldBeNull();
```

Comment:
The behavior here was inconsistent between the fluent and non-fluent APIs. The former returned a default value and the latter threw (per this test). In my uses I've always wanted the default value vs an error but I defer to you for which makes the most sense